Gobabygo/Gobabygo.ino

```diff
@@ -1,27 +1,28 @@
 const int joyStickXPin (0);
 const int joyStickYPin (1);
 const int rightMotorPin (6);
-const int leftMotorPin (4);
+const int leftMotorPin (5);
 const float deadzone = 0.1;
 
 unsigned long currentTime = 0;
 unsigned long previousTime = 0;
 
-float joyXVal = 0;
-float scaledJoyXVal = 0.0;
-int joyYVal = 0;
-float scaledJoyYVal = 0.0;
-int ledVal = 0;
-float rightMotorVal = 0.0;
-float leftMotorVal = 0.0;
+float joyXVal;
+float scaledJoyXVal;
+int joyYVal;
+float scaledJoyYVal;
+float rightMotorVal;
+float leftMotorVal;
+float scaler;
```

Comment:
For this group of variables I think it would be best to declare them local to your loop() or correctForDeadzone() methods (whichever it relevant) as its best to give variables the minimum scope needed. For example if a variable is only used in a certain method or its value does not need to persist across method calls its best to limit its scope to a specific method where it is used.